tests/acceptance/features/webUIFiles/batchAction.feature

```diff
@@ -0,0 +1,62 @@
+Feature: Visibility of the batch actions menu
+  As a user
+  I want to see the batch action menu options
+  So that the batch action menu options of the selected file or folder are visible to me
+
+
+  Background:
+    Given these users have been created with default attributes and without skeleton files:
+      | username |
+      | Alice    |
+      | Brian    |
+    And user "Alice" has created folder "simple-folder"
+
+
+  Scenario: View batch action menu for a folder on the favorites page
+    Given user "Alice" has logged in using the webUI
+    And user "Alice" has favorited element "simple-folder"
+    And the user has browsed to the favorites page
+    When the user selects the folder "simple-folder" using the webUI
+    Then the batch action button should not be visible on the favorites page
+
+
+  Scenario: View batch action menu for a folder on the shared with me page
+    Given user "Alice" has shared folder "simple-folder" with user "Brian"
+    And user "Brian" has logged in using the webUI
+    And the user has browsed to the shared-with-me page
+    When the user selects the folder "simple-folder" using the webUI
+    Then the decline action button should be visible on the shared-with-me page
+
+
+  Scenario: View batch action menu for a folder on the shared with others page
+    Given user "Alice" has logged in using the webUI
+    And user "Alice" has shared folder "simple-folder" with user "Brian"
+    And the user has browsed to the shared-with-others page
+    When the user selects the folder "simple-folder" using the webUI
+    Then the delete action button should be not visible on the shared-with-others page
```

Comment:
seems very similar. can we use a scenario outline here?

Reply:
it looks little bit similiar but steps and its order is not same in all scenario. So, i think scenario outline doesn't fit for this.